### DISCUSSION TITLE
Netplay: Sending configurations and metadata between players pre/post match

### DISCUSSION THREAD

**Discussion by assemblaj** [ikemen-engine/Ikemen-GO#1679] [Engine development]
We already send non-input data prior to the match for synchronization purposes.   Sending other data before/after the match could be very useful.  Could also use this to compare certain configs and see if they're compatible, or for things like showing the players' display names.  Might be worth looking into.   I don't have an opinions on implementation for this at the moment, let me know your thoughts.

**Comment by potsmugen** [comment]
K4thos has already implemented the configuration sync, which was sorely needed yeah.
